MediaBrowser.Controller/Providers/IDirectoryService.cs

```diff
@@ -16,5 +16,12 @@ public interface IDirectoryService
         IReadOnlyList<string> GetFilePaths(string path);
 
         IReadOnlyList<string> GetFilePaths(string path, bool clearCache, bool sort = false);
+
+        /// <summary>
+        /// Does the path exist.
+        /// </summary>
+        /// <param name="path">The path.</param>
+        /// <returns>Whether the path exists.</returns>
+        bool PathExists(string path);
```

Comment:
Should we not simply call it `Exists`? I'm not at all keen on the `IDirectoryService` interface, but I see the benefits when it comes to testing.